.dotnet.azure/src/Custom/Assistants/AzureAssistantsPageEnumerator.cs

```diff
@@ -0,0 +1,46 @@
+using System.ClientModel;
+using System.ClientModel.Primitives;
+
+#nullable enable
+
+namespace Azure.AI.OpenAI.Assistants;
+
+internal partial class AzureAssistantsPageEnumerator : AssistantsPageEnumerator
```

Comment:
Are we planning to remove InternalsVisibleTo at some point in the future? If we did, this wouldn't be able to inherit, but I don't see an issue with generating the two types separately without inheritance in that case and using generation to ensure consistency.

Reply:
Maybe someday; agreed that we'll need to figure it out if that day arrives, but you're right that there are viable ways to go. This is low on the worry list :)